content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx

```diff
@@ -1,17 +1,41 @@
 ---
 title: 'Examples'
-metaTitle: 'Prisma Client Extensions | Examples'
-metaDescription: 'Extend the functionality of Prisma Client: Examples'
+metaTitle: 'Prisma Client extensions | Prisma- & Community-built extensions and examples'
+metaDescription: 'Prisma Client extensions | Prisma- & Community-built extensions and examples'
 ---
 
-<TopBlock>
+## Extensions made by Prisma
 
-The extensions are provided as examples only, and without warranty. We recommend using the examples as a source of inspiration for building your own extensions.
+The following is a list of extensions we've built at Prisma:
 
-</TopBlock>
+| Extension                                                                              | Description                                                                                                                                              |
+| :------------------------------------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `@prisma/extension-accelerate`                                                         | Enables [Accelerate](https://www.prisma.io/accelerate), a global database cache available in 300+ locations with built-in connection pooling             |
+| `@prisma/extension-pulse`                                                              | Enables [Pulse](https://www.prisma.io/pulse), a service that captures change events from your database and delivers them instantly to your applications. |
+| [`@prisma/extension-read-replicas`](https://github.com/prisma/extension-read-replicas) | Adds read replica support to Prisma Client                                                                                                               |
+
+## Extensions made by Prisma's community
+
+The following is a list of extensions created by the community. If you want to create your own package, refer to the [Shared Prisma Client extensions](./shared-extensions) documentation.
+
+| Extension                                                                                      | Description                                                                                                                                    |
+| :--------------------------------------------------------------------------------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------- |
+| [`prisma-extension-supabase-rls`](https://github.com/dthyresson/prisma-extension-supabase-rls) | Adds support Supabase Row Level Security with Prisma                                                                                           |
```

Comment:
I'd suggest making the descriptions more consistent. Also "Adds support Supabase Row Level Security with Prisma    " has a typo, missing a "for" I think, "add support for".

Reply:
```suggestion
| [`prisma-extension-supabase-rls`](https://github.com/dthyresson/prisma-extension-supabase-rls) | Adds support for Supabase Row Level Security with Prisma                                                                                           |
```